model/Export/Packager/ExportZipPackager.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+namespace oat\taoSync\model\Export\Packager;
+
+
+use oat\oatbox\service\ConfigurableService;
+use oat\taoSync\model\Exception\SyncExportException;
+
+class ExportZipPackager extends ConfigurableService implements ExportPackagerInterface
```

Comment:
Can you please move this service out of `Export` namespace and also rename it to something like `PackagerService`. I would like to extend it's interface with unpack method in order to use it for importing as well.

Reply:
Moved the Packager to separate namespace as suggested